app/src/main/kotlin/com/wire/android/ui/authentication/create/CreateAccountFlowType.kt

```diff
@@ -0,0 +1,30 @@
+package com.wire.android.ui.authentication.create
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import com.wire.android.R
+
+enum class CreateAccountFlowType(
+    val routeArg: String = "",
+    @StringRes val titleResId: Int,
+    @StringRes val summaryTextResId: Int = 0,
+    @DrawableRes val summaryIconResId: Int = 0,
```

Comment:
Having initial values here is unitary + calling stringResource({ResIdThatDoesNotExist}) will throw an exception (I think 😅)
```suggestion
    val routeArg: String,
    @StringRes val titleResId: Int,
    @StringRes val summaryTextResId: Int,
    @DrawableRes val summaryIconResId: Int
```

Reply:
Right, fixed it 🚀